ballerina/Module.md

```diff
@@ -0,0 +1,100 @@
+## Overview
```

Comment:
We need to use observe module's `Module.md` (this seems to be from `ballerina/http`)

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/files#diff-54c2a1c9d62367d38139ae3d5d8200e06bc932ab7905f7e666d2c895382b2b81